compendium/DeclarativeServices/test/TestExceptionOnConstructionFailure.cpp

```diff
@@ -0,0 +1,148 @@
+/*=============================================================================
```

Comment:
I'd suggest naming this file similarly to the test fixture class; TestFailedBoundServiceActivation.cpp